examples/all-clusters-app/all-clusters-common/include/mode-select-delegates.h

```diff
@@ -0,0 +1,167 @@
+/*
+*
+*    Copyright (c) 2023 Project CHIP Authors
+*    All rights reserved.
+*
+*    Licensed under the Apache License, Version 2.0 (the "License");
+*    you may not use this file except in compliance with the License.
+*    You may obtain a copy of the License at
+*
+*        http://www.apache.org/licenses/LICENSE-2.0
+*
+*    Unless required by applicable law or agreed to in writing, software
+*    distributed under the License is distributed on an "AS IS" BASIS,
+*    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*    See the License for the specific language governing permissions and
+*    limitations under the License.
+*/
+
+#pragma once
+
+
+#include <app/clusters/mode-select-server/mode-select-delegate.h>
+#include <app/util/af.h>
+#include <app/util/config.h>
+#include <cstring>
+#include <utility>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+
+// Defining mode values as constants
+namespace ModeSelect {
```

Comment:
Why are we having this const mode values for ModeSelect? Shouldn't this be somehow manufacturer specific and these values should be set by manufacturer without modifying source code?

Reply:
This is set in a specific example app.  It should be set in app code however the app decides to set it.  I really hope no manufacturer is shipping all-clusters-app without modifying its source code....